README.md

```diff
@@ -9,8 +9,13 @@ Hint: To make merge sort in-place, think about what happens during the merge --
 where are elements moved to and from? To make it iterative, think about the
 part of the array each recursive call considers.
 
+I used the slides and video from class to look at how merge works and how it sorts. The TA helped me in office hours with runtime analysis.
+I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice.
+
 ## Runtime Analysis
 
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+In my code, I include 4 for loops, which would be $O(n^4)$, this would be the overall worst case since it runs over each element multiple times. So, the worst runtime would be ∈ $Θ(n^4)$, which is much worse than mergesort recursively and not in place.
```

Comment:
...but not each of the loop iterates over all the elements.

Reply:
Sorry, I'm still working on this.

Reply:
I realize my analysis was wrong, my first couple for loops are for separating the array in sections, which does not go through the data.